lib/concurrent-ruby/concurrent/atomic/mutex_semaphore.rb

```diff
@@ -23,8 +23,17 @@ def acquire(permits = 1)
 
       synchronize do
         try_acquire_timed(permits, nil)
-        nil
```

Comment:
I haven't worked with `synchronize` before, so I'm not sure if there was a reason this was in the same block. Didn't seem like it.